From 006b123a801afab44a9e1a3d6e2ff5e1c6415362 Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Wed, 28 Aug 2013 17:15:03 +0200
Subject: [PATCH] os: move <arpa/inet.h> for any !win32 system

It is needed in IPv6 configurations (for inet_pton) also when
SIOCGIFCONF is not defined.

Signed-off-by: Pino Toscano <toscano.pino@tiscali.it>
---
 os/access.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/os/access.c b/os/access.c
index 88a44d9..6d991b3 100644
--- a/os/access.c
+++ b/os/access.c
@@ -163,6 +163,10 @@ SOFTWARE.
 /* #endif */
 #endif
 
+#if defined(IPv6) && defined(AF_INET6)
+#include <arpa/inet.h>
+#endif
+
 #endif                          /* WIN32 */
 
 #define X_INCLUDE_NETDB_H
@@ -461,10 +465,6 @@ DefineSelf(int fd)
 #endif
 
 #if defined(IPv6) && defined(AF_INET6)
-#include <arpa/inet.h>
-#endif
-
-#if defined(IPv6) && defined(AF_INET6)
 static void
 in6_fillscopeid(struct sockaddr_in6 *sin6)
 {
-- 
1.7.10.4

